### PR TITLE
create readme file by answering some question

### PR DESCRIPTION
**Pull request by vivek80801**
## Assigned Issue Number ? { if does not exist then create one }
#38 

## Language Opted ?
Javascript

## Number of files added or updated ?
3

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've referred the correct issue number.
- [x] I've fill up this entire template correctly.

### CODE REVIEW

File: Automations/createReadmeFile/index.js [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+      resolve();
+    });
+  });
+};
+
+const ask_additonal_package = () => {
+  return new Promise((resolve, reject) => {
+    rl.question("what is name of additional package? \n", (answer) => {
+      additionalPackages.push(answer);
+      console.log(`\tName of additional package is ${answer}\n`)
```
Comment (Arsenic-ATG) [inline Automations/createReadmeFile/index.js:75]:
```suggestion
      console.log(`\tName of additional package is ${answer}\n`);
```

File: Automations/createReadmeFile/index.js [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+      console.log(`\tyour programming language is "${programmingLanguage}"\n`);
+      resolve();
+    });
+  });
+};
+
+const ask_additonal_packages = () => {
+  return new Promise((resolve, reject) => {
+    rl.question("Do you project need additional packages ? \n(write only 'yes' or 'no' in lowercase otherwise this program will fail.)\n", (answer) => {
+      isAdditionalPackages = answer === "yes" ? true : answer === "no" ? false : false
```
Comment (Arsenic-ATG) [inline Automations/createReadmeFile/index.js:64]:
```suggestion
      isAdditionalPackages = answer === "yes" ? true : answer === "no" ? false : false ;
```

File: Automations/createReadmeFile/index.js [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+      resolve();
+    });
+  });
+};
+
+const ask_things_needed_to_run = () => {
+  return new Promise((resolve, reject) => {
+    rl.question("what is name of thing need to run? \n", (answer) => {
+      thingsNeedToRun.push(answer)
+      console.log(`\tName of thing is ${answer}\n`)
```
Comment (Arsenic-ATG) [inline Automations/createReadmeFile/index.js:105]:
```suggestion
      thingsNeedToRun.push(answer);
      console.log(`\tName of thing is ${answer}\n`);
```

File: Automations/createReadmeFile/index.js [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+      numberOfSteps = answer;
+      console.log(`\tThere are ${numberOfSteps} steps to run project.\n`);
+      resolve();
+    });
+  });
+};
+const ask_step = () => {
+  return new Promise((resolve, reject) => {
+    rl.question("what is step to run the project? \n", (answer) => {
+      stepsToRun.push(answer)
```
Comment (Arsenic-ATG) [inline Automations/createReadmeFile/index.js:140]:
```suggestion
      stepsToRun.push(answer);
```

File: Automations/createReadmeFile/index.js [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+      console.log(`\tStep: ${answer}.\n`);
+      resolve();
+    });
+  });
+};
+
+(async function () {
+  try {
+    console.log("This automation only work for javascript and python.")
```
Comment (Arsenic-ATG) [inline Automations/createReadmeFile/index.js:149]:
```suggestion
    console.log("This automation only work for javascript and python.");
```

File: Automations/createReadmeFile/index.js [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+### How to use this program ? 💻
+
+${stepsToRun.map((st) => `- ${st}
+`).join("")}
+
+### Made By :-
+[${nameOfCreator}](${linkOfProfileOfCreator} "Go to profile of creator")
+  ` : ""}
+" > README.md`);
+    console.log("your file is created.")
```
Comment (Arsenic-ATG) [inline Automations/createReadmeFile/index.js:228]:
```suggestion
    console.log("your file is created.");
```

**Review by Arsenic-ATG — CHANGES_REQUESTED** [review]
Although not code breaking changes, but you missed some semicolons. 

I know that Js supports ASI ( automatic semicolon insertion ) but these are the ones pointed out by my [linting](https://en.wikipedia.org/wiki/Lint_%28software%29) software, so I thought it would be better to point them out, just be on a safer side.

**Comment by vivek80801** [comment]
@Arsenic-ATG  

Thanks for your code review and feedback.

I don't used linter in this project because it will increase complexity unnecessarily. I think, we don't need for this project. We can create a `github action` which will fix linting errors. `Github actions` are free to use for public repos. For ```Javascript``` , we use `eslint` to lint our code and `prettier` as code formatter. Both can be intergrated with `github actions`. I don't know, What is used in python world for liniting.

**Review by Arsenic-ATG — APPROVED** [review]

**Pull request merged**